packages/decision-tree/src/components/Question/questionTypes/Numeric/Numeric.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import { type QuestionProps } from '@components/Question/types';
+import { Box, MaskInput, Flex } from '@cmpsr/components';
+import { QuestionTitle } from '../../components/QuestionTitle';
+import { NumericQuestion } from './types';
+import { HandleAnswersActions } from '@hooks';
+
+export const Numeric: FC<QuestionProps> = ({ data, answersDispatch }) => {
+  const { question, tooltip, id, placeholder, trailingMask } = data as NumericQuestion;
+
+  return (
+    <Box>
```

Comment:
ditto

Reply:
it's for the stylings I think, in the Next PR it will look different